docs/get-started/getting-started-workload-qualification.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: page
+title: Workload Qualification
+nav_order: 7
+parent: Getting-Started
+---
+# Getting Started on Spark workload qualification
+
+The RAPIDS Accelerator for Apache Spark does not support all the features which are supported in Apache Spark.
+If you plan to convert existing Spark workload from CPU to GPU, it is highly recommended that you do gap analysis 
+to figure out if there are any unsupported features such as functions, expressions, data types, data formats.
+After that we will know which workload is more suitable to GPU, which is called "workload qualification" here.
+
+If there are certain operators which can not run on GPU due to the current limitations, they will fallback to CPU mode.
+As a result, it may incur some performance overhead because of host memory <=> GPU memory transfer. 
+
+This article is to help you get familiar with the best practice and different tools we provide on how to do gap analysis 
+and workload qualification.
+
+## 1. Qualification and Profiling tool
+
+### Requirement
+
+- Access to Spark event logs from Spark 2.x or 3.x
+- Spark 3.0.1+ jars
+- `rapids-4-spark-tools` jar
+
+### How to use
+
+If you have old Spark event logs from Spark 2.x or 3.x, you can use [Qualification tool](../spark-qualification-tool.md) 
```

Comment:
perhaps just old to be something more along the lines of Spark event logs from prior runs of the application. the event logs used might only be minutes "old".

Add the to the part: "you can use the Qualification tool]

Reply:
Changed to:
```
If you have Spark event logs from prior runs of the applications on Spark 2.x or 3.x, you can use the
[Qualification tool]
```
How do you think?

Reply:
that works